src/User/models.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class User(models.Model):
+    name = models.CharField(max_length=100)
+    email = models.EmailField()
```

Comment:
Email will be unique?

Reply:
Changes has been made, Email field cannot be repeated anymore.